utils.py

```diff
@@ -262,11 +263,19 @@ def get_next_clone_name(all_names, basename, name_suffix='', ts=False):
 
 def is_libvirtd_up():
     """
-    Checks if libvirtd.service is up.
+    Checks if libvirt is up.
     """
-    cmd = ['systemctl', 'is-active', 'libvirtd.service']
-    output, error, rc = run_command(cmd, silent=True)
-    return True if output == 'active\n' else False
+    isSocket = False
+    paths = ['/var/run/libvirt/libvirt-sock', '/usr/local/var/run/libvirt/libvirt-sock']
+    for path in paths:
```

Comment:
Instead of hard coding the directory paths, we could use automake macros to get its value. So it will be only one path and we don't need to `for` loop below.
You can check how we do it for `pythondir` and use `runstatedir`. See more details in https://www.gnu.org/prep/standards/html_node/Directory-Variables.html

The code would be to set `@runstatedir@` in config.py.in to variable and use it in here:

```
if os.stat(os.path.join(config.runstatedir, 'libvirt/libvirt-sock'):
     # do work
```